### PR TITLE
Issues/cleanup config

### PR DESCRIPTION
**Pull request by pmlopes**
As a follow up from todays call here is a clean up to the config that would allow both Web and API to co-exist.

Main changes are:

* there are no more environment specific endpoints but a single `secured-booster-endpoint`
* there are no more environment specific roles but a single `booster-admin`

Web apps must authenticate against `demoapp` as documented in the story docs as this client allows the usage of `authentication code` flow.

APIs should talk to the `secured-booster-endpoint` client as it is a `bearer only` client that does not do authN just AuthZ.

### CODE REVIEW

File: src/main/resources/master.json [hunk 1]
```diff
@@ -1,1597 +1,1551 @@
...
+          "id": "9ef8c962-b41a-4c6c-8e86-a8c22bde7808",
+          "name": "create-realm",
+          "description": "${role_create-realm}",
+          "scopeParamRequired": false,
+          "composite": false
+        },
+        {
+          "id": "0a75851e-63f0-45d2-8e5e-d43218aa9a08",
+          "name": "booster-admin",
+          "description": "A Vert.X administrator role",
```
Comment (Ladicek) [inline src/main/resources/master.json:76]: Vert.x? :-)

File: src/main/resources/master.json [hunk 2]
```diff
@@ -1,1597 +1,1551 @@
...
         }
+      ],
+      "client": {
+        "demoapp": [],
+        "security-admin-console": [],
+        "secured-booster-endpoint": [
+          {
+            "id": "1f4bea15-a2b3-4a34-afc9-c0e264be8136",
+            "name": "booster-admin",
+            "description": "A Vert.x administrator client",
```
Comment (Ladicek) [inline src/main/resources/master.json:95]: Vert.x? :-)

**Comment by pmlopes** [comment]
Well spoted, i've just pushed an update to the labels. no more vert.x references

**Comment by Ladicek** [comment]
Thanks @pmlopes!

**Pull request merged**